ui/package.json

```diff
@@ -3,8 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "typescript": ">=1.5.0-beta",
+    "chai": "^3.2.0",
```

Comment:
http://stackoverflow.com/questions/22343224/difference-between-tilde-and-caret-in-package-json

It will accept higher minor versions.